package.json

```diff
@@ -20,7 +20,7 @@
     "build": "rimraf dist && yarn lint && tsc --project ./tsconfig.json",
     "lint": "yarn eslint src/**",
     "test": "NODE_OPTIONS=--loader=ts-node/esm jasmine --config=jasmine.json",
-    "integration-test": "docker compose up -d && node ./scripts/init_onchain_data.js && NODE_OPTIONS=--loader=ts-node/esm jasmine --config=jasmine-integration.json ; docker compose down",
+    "integration-test": "node ./scripts/init_onchain_data.js && NODE_OPTIONS=--loader=ts-node/esm jasmine --config=jasmine-integration.json",
```

Comment:
We'll probably have to put some explanation in the README about this.

Reply:
Or we find an easy way to launch a collator node using zombienet , that's the same problem as `node-api`.